kong/request.go

```diff
@@ -9,13 +9,9 @@ import (
 	"github.com/google/go-querystring/query"
 )
 
-// NewRequest creates a request based on the inputs.
-// endpoint should be relative to the baseURL specified during
-// client creation.
-// body is always marshaled into JSON.
-func (c *Client) NewRequest(method, endpoint string, qs interface{},
+//NewRequestRaw creates a request based on the inputs.
+func (c *Client) NewRequestRaw(method, baseURL string, endpoint string, qs interface{},
```

Comment:
What if we rename baseURL to `url`? 
I _think_ it makes the signature a bit more clear.

Reply:
Using url would mean using the full one, isn't it?
As it's always concatenated with the endpoint parameter, wouldn't it it lead to confusion?

Reply:
I misunderstood. @mflendrich Should we name these as `NewWorkspacedRequest` `NewRequest` and then appropriate docs comment? The current API in this patch is not very easy to grok.